FireboltNETSDK/Client/FireboltParameter.cs

```diff
@@ -276,32 +222,19 @@ internal static string TrimParameterName(string parameterName)
             return parameterName;
         }
 
-        private class ParameterColumnTypeDescriptorAdapter
+        private static int GetSize(object? value)
         {
-            private readonly FireboltParameter _parameter;
-
-            public string ColumnName => _parameter.Id;
-
-            public FireboltDbType? FireboltDbType => _parameter._forcedType;
-
-            public Type ValueType => _parameter?.Value?.GetType() ?? typeof(DBNull);
-
-            public bool? IsNullable => _parameter._forcedNullable;
-
-            public int Size => _parameter._size;
-
-            public byte? Precision => _parameter._forcedPrecision;
-
-            public byte? Scale => _parameter._forcedScale;
-
-            public TimeZoneInfo? TimeZone => _parameter.TimeZone;
-
-            public int? ArrayRank => _parameter._forcedArrayRank;
-
-            public ParameterColumnTypeDescriptorAdapter(FireboltParameter parameter)
+            if (value == null)
             {
-                _parameter = parameter;
+                return 0;
             }
+            //TODO: improve this: make size discovery dependent on value type
+            return (value.ToString() ?? string.Empty).Length;
```

Comment:
I feel like this could be potentially misleading. Should we just throw an error for the time being if getsize is called on a non-null value?

Reply:
I think implementation of this function is reasonable. If value is null it returns 0. Otherwise it returns size of its string representation. But if the string representation is null itself it is replaced by empty string and therefore 0 will be returned. 

So, this function can be re-written as following:

```
if (value == null)
{
    return 0;
}
string? str value.ToString();
return str == null ? 0 : str.Length;
```

I also added comment 
```
//TODO: improve this: make size discovery dependent on value type
```
that means that I am planning to improve this code.

Reply:
I understand what it does and what TODO is. I don't agree with the current implementation. Size is supposed to represent a number of bytes for non-string types ([doc](https://learn.microsoft.com/en-us/dotnet/api/system.data.common.dbparameter.size?view=net-7.0#remarks)) so returning a length of their string representation is misleading. We can't just change logic on a whim, even if we add comments.

Reply:
implemented